.github/workflows/ci.yml

```diff
@@ -42,9 +42,6 @@ jobs:
         ember-version:
           [
             # The test suite only support Ember 3.x
-            # ember-lts-2.8,
-            # ember-lts-2.12,
-            # ember-lts-2.18,
```

Comment:
Can we leave this? I'd like to express the ambition that the suite works with them.

Having poked at dependency issues for a few days now, I think I also have an idea as to what could be breaking and it may just be in the test suite.